### DISCUSSION TITLE
Lesson 1: Clarification on Etherscan vs Arbiscan QR code challenge

### DISCUSSION THREAD

**Discussion by spencerj411** [Cyfrin/foundry-full-course-cu#1357] [Q&A] [answered]
Hi, all! I was able to solve the challenge! However, a few questions:
- I was only able to do it with the etherscan link, not with the arbiscan link. Why is this the case? More specifically, I was not able to connect my MM wallet via arbiscan but I was able to do so on etherscan and thus, was able to call the function. Is it because the contract on arbiscan is on a different blockchain? 
- How come when I copy the contract address from the arbiscan link into etherscan ([https://sepolia.etherscan.io/address/0x2e99a4ca9c85383ccb54769adb3837437e296479](https://sepolia.etherscan.io/address/0x2e99a4ca9c85383ccb54769adb3837437e296479)) it doesn't show as a contract on etherscan?

In other words, I'd like to know what the difference is between the two links. Thanks in advance!

**Comment by spencerj411** [comment, marked as answer]
The answer is the etherscan link is for the Sepolia blockchain and the arbiscan link is for the Arbritrum blockchain.
